test-integration/Microsoft.Azure.WebJobs.Extensions.OpenApi.TestApp/Post_ApplicationJson_DecimalType_HttpTrigger.cs

```diff
@@ -18,6 +18,7 @@ public class Post_ApplicationJson_DecimalType_HttpTrigger
     {
         [FunctionName(nameof(Post_ApplicationJson_DecimalType_HttpTrigger))]
         [OpenApiOperation(operationId: nameof(Post_ApplicationJson_DecimalType_HttpTrigger.Post_ApplicationJson_DecimalType), tags: new[] { "demical" })]
+        [OpenApiRequestBody(contentType: "application/octet-stream", bodyType: typeof(decimal), Required = true, Description = "The OK response")]
```

Comment:
contentType does not match with your test DataRow.
I think contentType should be "text/plain", not application/octet-stream

Reply:
Thanks, I'll give it a try